src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -961,38 +1008,44 @@ private static bool HasExplicitNullableConversion(TypeSymbol source, TypeSymbol
 
             if (!source.IsNullableType() && !destination.IsNullableType())
             {
-                return false;
+                return Conversion.NoConversion;
             }
 
-            TypeSymbol sourceUnderlying = source.StrippedType();
-            TypeSymbol destinationUnderlying = destination.StrippedType();
+            TypeSymbol unwrappedSource = source.StrippedType();
+            TypeSymbol unwrappedDestination = destination.StrippedType();
 
-            if (HasIdentityConversion(sourceUnderlying, destinationUnderlying))
+            if (HasIdentityConversion(unwrappedSource, unwrappedDestination))
             {
-                return true;
+                return Conversion.ExplicitNullable;
```

Comment:
And in some cases below?

---
In reply to: [67725469](https://github.com/dotnet/roslyn/pull/12068#discussion_r67725469) [](ancestors = 67725469)